README.md

```diff
@@ -19,24 +19,40 @@ In order to access the development branch on contentful during development set t
 ENV=dev yarn develop
 ```
 
+and your .env file to match the contentful keys and tokens
+
+```
+CONTENTFUL_TOKEN=(see in contentful/settings/API keys)
+CONTENTFUL_SPACE=(see in contentful/settings/API keys)
+MEETUP_KEY=(see in contentful/settings/API keys)
+GATSBY_ENVIRONMENT="development"
+```
+
 We have some docs to make it easier to get you started:
 
 - [Scripts Overview](./docs/scripts.md)
 
-
-## Tech Used
+## Tech used
 
 - [Yarn](https://yarnpkg.com)
 - [Gatsby](https://www.gatsbyjs.org/docs/)
 - [React](https://reactjs.org)
 - [Styled Components](https://styled-components.com)
 - [Storybook](https://storybook.js.org/)
 
+## Tests stack
+
+- [Jest](https://jestjs.io/)
+- [Lighthouse](https://www.gatsbyjs.org/docs/audit-with-lighthouse/)
+
+Lighthouse expects you to have your app built locally in and served from the /public directory.
+As a result make sure to run `npm run build` before running the Lighthouse tests. Depending on your environment you may on occasion also need to serve manually with `serve public`.
```

Comment:
I think the last sentence (about serving it manually) may be a bit confusing.. I just asked you to run `serve` on our call for me to understand if the build was OK, but out of context it can be hard to understand

Maybe add something like "If you want to check the ouput of the build, run `gatsby serve` ?

Reply:
Yes. It's not only for checking. I will rephrase.